### PR TITLE
fix: docs

### PR DESCRIPTION
**Pull request by PalmDevs**
Fixes the documentation.

**CURRENTLY A DRAFT!**

### CODE REVIEW

**Comment by PalmDevs** [comment]
semantic pr has skill issues

**Comment by ILoveOpenSourceAppications** [comment]
What is this tbh?

File: docs/0_setting-up.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# 👶 Setting up
+# 🔨 Setting up
 ## 📝 Requirements
 - Android 8+
-- ARM64 device
+- ARM64 device with ARMv8 *(Binder64 devices will NOT work)*
 
 ## ⬇️ Installation
-- Download and install the [latest ReVanced Manager APK](https://github.com/revanced/revanced-manager/releases/latest)
+- Download and install the [latest ReVanced Manager](https://github.com/revanced/revanced-manager/releases/latest)
```
Comment (oSumAtrIX) [inline docs/0_setting-up.md:7]:
```suggestion
- Download and install [ReVanced Manager](https://github.com/revanced/revanced-manager/releases/latest)
```

File: docs/0_setting-up.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# 👶 Setting up
+# 🔨 Setting up
 ## 📝 Requirements
 - Android 8+
-- ARM64 device
+- ARM64 device with ARMv8 *(Binder64 devices will NOT work)*
```
Comment (oSumAtrIX) [inline docs/0_setting-up.md:4]: This is too technical.

Comment (PalmDevs) [reply docs/0_setting-up.md:4]: Isn't that what documentation are supposed to be? It has to contain all the information if possible.

**Comment by PalmDevs** [comment]
> What is this tbh?

A PR to fix documentation.

**Pull request closed without merging**